docs/topics/managedcluster.md

```diff
@@ -0,0 +1,147 @@
+# Managed Clusters (AKS)
+- **Feature status:** Experimental
+- **Feature gate:** AKS=true,MachinePool=true
+
+Cluster API Provider Azure (CAPZ) experimentally supports managing Azure
+Kubernetes Service (AKS) clusters. CAPZ implements this with three
+custom resources:
+- AzureManagedControlPlane
+- AzureManagedCluster
+- AzureManagedMachinePool
+
+The combination of AzureManagedControlPlane/AzureManagedCluster
+corresponds to provisioning an AKS cluster. AzureManagedMachinePool
+corresponds one-to-one with AKS node pools. This also means that
+creating an AzureManagedControlPlane requires defining the default
+machine pool, since AKS requires at least one system pool at creation
+time.
+
+## Deploy with clusterctl
+
+A clusterctl flavor exists to deploy an AKS cluster with CAPZ. This
+flavor requires the following environment variables to be set before
+executing clusterctl.
+
+```bash
+# Kubernetes values
+export CLUSTER_NAME="foobar"
+export WORKER_MACHINE_COUNT=2
+export KUBERNETES_VERSION="1.17.4" # NOTE: no preceding "v", unlike normal for CAPI.
+
+# Azure values
+export AZURE_LOCATION="southcentralus"
+export AZURE_RESOURCE_GROUP="${CLUSTER_NAME}"
+export AZURE_SUBSCRIPTION_ID="$(cat ~/sp.json | jq -r .subscriptionId | tr -d '\n')"
+export AZURE_NODE_MACHINE_TYPE="Standard_D8s_v3"
```

Comment:
was it intentional to specify this particular VM size? or just an example? the rest of the capz docs use `Standard_D2s_v3`.